dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -169,11 +169,13 @@ bool FramesDecoder::ReadRegularFrame(uint8_t *data, bool copy_to_output) {
     }
 
     if (!copy_to_output) {
+      ++current_frame_;
```

Comment:
In case of `if (ret == AVERROR_EOF) {` maybe it should set `current_frame_ = -1`?

Reply:
Not really. When this condition triggers it means that we no longer have packages to read. But we still might have frames in internal FFMPEG buffers.
We set it when we have flushed all frames from FFMPEG internals.